### PR TITLE
Add tests for helpers and zadara helpers

### PR DESCRIPTION
**Pull request by loicgasser**
This PR introduce some unit tests for our helpers. I'll try to add more and more of those kind of tests since they are fast to excecute and will help us organize the code.

Side note:
`apache_base_path` is the same for all envs is already present in `buildout.cfg`. As a result we can remove it.

### CODE REVIEW

File: chsdi/tests/functional/test_zadara_helpers.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    def test_find_files(self):
+        import os
+        request = testing.DummyRequest()
+        request.host = 'api3.geo.admin.ch'
+        request.scheme = 'http'
+        request.registry.settings = {}
+        request.registry.settings['apache_base_path'] = 'main'
+        request.registry.settings['zadara_dir'] = '/var/local/cartoweb/downloads/'
+        layerBodId = 'ch.swisstopo.geologie-gisgeol'
+        fileName = os.listdir(request.registry.settings['zadara_dir'] + layerBodId)[0]
```
Comment (gjn) [inline chsdi/tests/functional/test_zadara_helpers.py]: Maybe add a test to check if find_files returns empty array?

**Comment by gjn** [comment]
LGTM. Thanks for the tests and improvements.
